4.4.2/style.css

```diff
@@ -0,0 +1,77 @@
+
+.facebook-text {
+  width: 200px;
+  padding: 20px;
+  padding-top: 0;
+  font-family: Fira Code;
+  font-size: 1em;
+  color: white;
+}
+
+.instagram-text {
+  width: 200px;
+  padding: 20px;
+  padding-top: 0;
+  font-family: sans-serif;
+  font-size: 1em;
+  color: white;
+}
+
+.twitter-text {
+  width: 200px;
+  padding: 20px;
+  padding-top: 0;
+  font-family: monospace;
+  font-size: 1em;
+  color: white;
+}
+
+.facebook-container {
+  background-color: #003375;
+}
+
+.instagram-container {
+  background-color: #d4aa27;
+}
+
+.twitter-container {
+  background-color: #0A74FF;
+}
+
+a:hover {
+    color: yellow;
+}
+.fa-facebook {
+  font-size: 3em;
+  margin-left: 40%;
+  margin-top: 33%;
+}
+
+.fa-twitter-square {
+  font-size: 3em;
+  margin-left: 40%;
+  margin-top: 18%;
+}
+
+.fa-instagram {
+  font-size: 3em;
+  margin-left: 40%;
+  margin-bottom: 10%;
+}
```

Comment:
The same for the icons, make one general class and if they have different properties apply a specific class, but also, in this case, the icons are not aligned (https://www.screencast.com/t/vq58l7eI)